src/pages/blog/starlingx-dual-stack-support.md

```diff
@@ -0,0 +1,267 @@
+---
+templateKey: blog-post
+title: Dual-Stack (IPv4/IPv6) Support in Starlingx
+author: Andre Kantek
+date: 2024-08-12T01:32:05.627Z
+category:
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+
+StarlingX, a cloud platform designed for edge computing, now supports dual-stack functionality on its platform networks (OAM, management, cluster-host, cluster-pod, cluster-service, admin, storage, and multicast), enabling it to operate with both IPv4 and IPv6 L3 protocols. This enhanced capability offers greater flexibility and scalability for network deployments. Each platform network can be associated to two pools (one IPv4 and another IPv6). The order it is done defines the primary and secondary pool for that network.
+
+The Kubernetes network can also receive dual-stack properties, allowing the pods to use IPv4 and/or IPv6 on its operations. It becomes available when the OAM, cluster-host, cluster-pod, and cluster-service networks are configured as dual-stack.
+
+# Key Features and Considerations
```

Comment:
This section looks and reads like a recap of the feature's documentation, which doesn't fit very well into a blog post. It would be better to remove this and add a pointer to the feature documentation in the 'Conclusion' section on the bottom of the article.

Reply:
done